charts/fission-all/templates/executor/deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
         prometheus.io/path: "/metrics"
         prometheus.io/port: "8080"
     spec:
+      {{- if .Values.executor.securityContext.enabled }}
+      securityContext: {{- omit .Values.executor.securityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
```

Comment:
Values from fetcher section are used below(line no 54) in executor container.
We will need to see how we can utilize fetcher security context here.